codegenerator/cli/templates/static/codegen/src/eventFetching/FetchState.res

```diff
@@ -15,6 +15,15 @@ type blockNumberAndLogIndex = {blockNumber: int, logIndex: int}
 
 type status = {mutable fetchingStateId: option<int>}
 
+type partitionKind =
+  | Wildcard
+  | Normal({
+      contractAddressMapping: ContractAddressingMap.mapping,
+      //Used to prune dynamic contract registrations in the event
+      //of a rollback.
+      dynamicContracts: array<TablesStatic.DynamicContractRegistry.t>,
+    })
```

Comment:
In the future, to support topic filtering for rpc, we'll need to have a partition per event.